src/hotspot/share/opto/loopPredicate.cpp

```diff
@@ -396,9 +395,13 @@ void PhaseIdealLoop::get_assertion_predicates(Node* predicate, Unique_Node_List&
 IfProjNode* PhaseIdealLoop::clone_assertion_predicate_for_unswitched_loops(Node* iff, IfProjNode* predicate,
                                                                            Deoptimization::DeoptReason reason,
                                                                            ParsePredicateSuccessProj* parse_predicate_proj) {
-  Node* bol = create_bool_from_template_assertion_predicate(iff, nullptr, nullptr, parse_predicate_proj);
+  Node* opaque4_node = iff->in(1);
```

Comment:
`assert(opaque4->Opcode() == Op_Opaque4, "must be Opaque4");`
from `create_bool_from_template_assertion_predicate` ?